src/GNNGraphs/GNNGraphs.jl

```diff
@@ -77,6 +77,7 @@ export add_nodes,
        to_bidirected,
        to_unidirected,
        random_walk_pe,
+       remove_nodes,
```

Comment:
the function should also be added to `docs/`

Reply:
just to confirm, you mean in `docs/src/assets/gnngraph.md` right?

Reply:
yes